frontend/src/component/incomingWebhooks/IncomingWebhooksModal/IncomingWebhooksModal.tsx

```diff
@@ -137,12 +151,16 @@ export const IncomingWebhooksModal = ({
             <FormTemplate
                 loading={loading}
                 modal
-                title={title}
+                title=''
```

Comment:
Could we make this property/parameter optional? Or did you want to show that it's intentionally omitted?

Reply:
Seems like it's already optional so we can just omit it. Nice catch! https://github.com/Unleash/unleash/pull/6317/commits/f35add8c1545efc11cba516126b63b8cd956f581